lib/modules/dosomething/dosomething_helpers/dosomething_helpers.share.inc

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * @file
+ * dosomething_helpers.share.inc
+ */
+
+/**
+ * Returns an empty page with javascript that closes the current window.
+ * Used as the redirect page for facebook shares that require a redirect uri.
+ * Allows us to close the share popup window when the user has finished sharing.
+ */
+function dosomething_helpers_share_redirect() {
+  return "<script>(function(){ window.close(); })();</script>";
+}
+
+/**
+ * Returns a facebook share url.
+ *
+ * @param  string $link The url to share.
+ * @param  string $type
+ *  The type of facebook share intent link to use.
+ *    - 'feed_dialog': Allows for customized stories, outside of what is set in the metatags.
+ * @param Array $custom_options
+ *    An array of customized options for the type of share being used.
+ *
+ * @return string   The facebook intent url.
+ */
+function dosomething_helpers_facebook_intent($link, $type = NULL, $custom_options = NULL) {
```

Comment:
Same for other functions... not necessary for merge or anything, but at first glance i wasn't sure what the INTENT of the function was (ha ha oh man its friday)